app/navigation/Navigation.scala

```diff
@@ -73,7 +73,8 @@ class Navigation @Inject()(dateTimeHelper: DateTimeHelper,
     AppealStartPage -> ((_, _, _) => routes.AppealStartController.onPageLoad()),
     OtherRelevantInformationPage -> ((_, _, _) => routes.OtherReasonController.onPageLoadForUploadEvidence(NormalMode)),
     CancelVATRegistrationPage -> ((answer, _, extraData) => routingForCancelVATRegistrationPage(answer, extraData)),
-    OtherPenaltiesForPeriodPage -> ((_, _, _) => routes.AppealStartController.onPageLoad())
+    OtherPenaltiesForPeriodPage -> ((_, _, _) => routes.AppealStartController.onPageLoad()),
+    YouHaveUploadedFilesPage -> ((_, _, _) => routes.OtherReasonController.onPageLoadForFirstFileUpload(NormalMode))
```

Comment:
This would cause the user to be redirected back to the first file upload page. Is this a placeholder?

Reply:
Removed the redundant Navigation